torchaudio/functional/filtering.py

```diff
@@ -83,14 +89,20 @@ def allpass_biquad(
         http://sox.sourceforge.net/sox.html
         https://www.w3.org/2011/audio/audio-eq-cookbook.html#APF
     """
+    dtype = waveform.dtype
+    device = waveform.device
+    central_freq = _float2Tensor(central_freq, dtype, device).squeeze()
+    Q = _float2Tensor(Q, dtype, device).squeeze()
```

Comment:
Why the call to squeeze here each time?

Reply:
To make sure that arguments pass to `biquad` in return statement are scalar tensors, so it can be jittable.
But I found that it's not necessary, cuz if user pass unit length tensor (like `torch.Tensor([1.])`) as input and use `torch.jit.script`, it will throw error in the first place before calling `biquad`.
I will remove these lines.